packages/icons/icons.json

```
@@ -1486,126 +1486,126 @@
     {
         "displayName": "Lightbulb",
         "iconName": "lightbulb",
-        "tags": "idea, glow, lamp",
+        "tags": "idea, glow, lamp, light",
         "group": "miscellaneous",
         "codepoint": 61995
     },
     {
         "displayName": "Glass",
         "iconName": "glass",
-        "tags": "glassware, drink",
+        "tags": "glassware, drink, cheers, cup, wine glass, wine, chalice, goblet",
         "group": "miscellaneous",
         "codepoint": 61905
     },
     {
         "displayName": "Tint",
         "iconName": "tint",
-        "tags": "drop, color, coloration, hue",
+        "tags": "drop, color, coloration, hue, tear, rain, water, rain drop, droplet",
         "group": "media",
         "codepoint": 62197
     },
     {
         "displayName": "Flash",
         "iconName": "flash",
-        "tags": "light, contrast, photograph, picture",
+        "tags": "light, contrast, photograph, picture, sun, star, illuminate, examine, brighten",
         "group": "media",
         "codepoint": 61872
     },
     {
         "displayName": "Font",
         "iconName": "font",
-        "tags": "scale, typography, size",
+        "tags": "scale, typography, size, a, letter, alphabet, letter a, area",
```

```suggestion
        "tags": "scale, typography, size, letter, alphabet",
```

"a" is not going to help anyone find an icon via search

---

agreed!